blockchain/contracts/HelloWorld.sol

```diff
@@ -29,16 +37,20 @@ contract HelloWorld {
         nftOwner = payable(0xC77787e364E3420c0609a249F18De47430900f0C);
```

Comment:
What is the meaning of this big magic number? Is it possible in Solidity to create a constant for it?

Reply:
It is not a big magic number but an address (to be more specific – Jannis' address). It is hardcoded for testing reasons since the contract is not yet deployed and we also didn't decide yet who the initial owner of the NFTs should be (the person who creates the drop? Or should the person who creates the drop specify this? etc.)

Reply:
Okay, so the meaning is actually: `uint256 jannisAddress = 0xC77787e364E3420c0609a249F18De47430900f0C;`

Reply:
No, it is not. The meaning is the current NFT owner is this address and can be payed. It used to be Nick's. As I said, it's only for testing.

Reply:
It's not forbidden to introduce variables for testing only ;-) . I think, it will also make review more easy.